src/core_modules/capture-core/components/D2Form/D2SectionFields.component.js

```diff
@@ -269,20 +278,27 @@ export class D2SectionFieldsComponent extends Component<Props> {
 
         this.buildRulesCompulsoryErrors();
 
+        const styles = getStyles();
```

Comment:
Can we remove this function call and just use the `style` object directly? To make it work without the function, you can change line [21](https://github.com/dhis2/capture-app/pull/3457/files#diff-4a065843059602ff4759ed922279b5fcfa54c3a562b3fbafbb23b5b5ec270c46R21) to:
```
constant styles = {
          horizontal section: {
              flexible screen',
              flexWrap: 'wrap',
              alignItems: 'flexible start',
          },
};
```

Thanks!

Reply:
Thank you, @simonadomnisoru!